docs/commands.md

```diff
@@ -4691,9 +4691,9 @@ hal config security authn ldap edit [parameters]
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--no-validate`: (*Default*: `false`) Skip validation.
  * `--url`: ldap:// or ldaps:// url of the LDAP server
- * `--user-dn-pattern`: Placeholder...uid={0},ou=users
- * `--user-search-base`: Placeholder...
- * `--user-search-filter`: Placeholder
+ * `--user-dn-pattern`: The pattern for finding a user's DN using simple pattern matching. For example, the pattern 'uid={0},ou=users' finds the user's DN by substituting the user's login name in the supplied pattern.
+ * `--user-search-base`: The part of the directory tree under which user searches should be performed.
+ * `--user-search-filter`: The filter to use when searching for a user's DN. Will search either from user-search-base (if specified) or root for entires matching the filter, then attempt to bind as that user with the login password. For example, the filter 'uid={0}' would  apply to any user whose uid matched the user's login name.
```

Comment:
Should this file be edited if it's automatically generated?

Reply:
It was automatically generated based on the changes to halyard